### ISSUE TITLE
No proofreading in doc comments in 2020.2

### ISSUE THREAD

**Issue by Undin** [intellij-rust/intellij-rust#5530]
## Environment

* **IntelliJ Rust plugin version:** 0.2.124.SNAPSHOT-202-dev
* **Rust toolchain version:** 1.46.0-nightly (0262de554 2020-06-07) x86_64-apple-darwin
* **IDE name and version:** CLion 2020.2 EAP (CL-202.5103.16)
* **Operating system:** macOS 10.13.6

## Problem description
There isn't the corresponding annotation in 2020.2
<img width="318" alt="Screen Shot 2020-06-08 at 16 35 36" src="https://user-images.githubusercontent.com/2539310/84036718-470e3480-a9a6-11ea-995e-b9f5cbd48c55.png">

Note, it exists in 2020.1
**UPD**: looks like bug is backported in 2020.1.2 because it doesn't work in 2020.1.2 as well

<img width="315" alt="Screen Shot 2020-06-08 at 16 36 41" src="https://user-images.githubusercontent.com/2539310/84036809-69a04d80-a9a6-11ea-954d-249a1bf3ebc8.png">

## Steps to reproduce
```rust
/// ```
/// let a = "There is two apples";
/// ```
fn foo() {

}
```

**Comment by Undin** [comment]
https://youtrack.jetbrains.com/issue/IDEA-244001

**Comment by Undin** [comment]
https://youtrack.jetbrains.com/issue/IDEA-244001 was fixed in 2020.3 so I think we can close this issue too
